### PR TITLE
ci: add 'one-approval' label on PR that has one approval

### PR DESCRIPTION
**Pull request by leseb**
If a PR has been approved already by one of the maintainers, the PR will be labeled with 'one-approval'. This is useful for maintainers to help them determine which PR is getting close to completion and thus prioritize the review.
With the help of filters, one can look for PRs that have one approval already.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
@russellb PTAL

**Comment by leseb** [comment]
Side note: the "Approved review" filter on PRs will only filter PRs that have two approvals (GitHub config?) so this will still be useful.

File: .github/mergify.yml
Comment (russellb) [inline .github/mergify.yml:183]:
```suggestion
      - "#approved-reviews-by!=1"
```

**Review by russellb — CHANGES_REQUESTED** [review]
I think the current config would lave the label on even after there's more than just one approval on it

File: .github/mergify.yml
Comment (leseb) [reply .github/mergify.yml:183]: It's a good point, thanks!

**Review by nathan-weinberg — APPROVED** [review]
I don't want to get too label-happy here but this does seem like it could be a cool addition - willing to give it a shot!

**Review by russellb — APPROVED** [review]

**Pull request merged**
